data/coalition/coalition jobs.txt

```diff
@@ -1322,3 +1322,432 @@ mission "Coalition: From Games 3"
 		payment
 		payment 25000
 		dialog phrase "coalition games dialog"
+
+
+mission "Hely Contain True"
+	job
+	repeat
+	name `Contain Resistance`
+	description `Fly to <destination> so that you and <bunks> other Heliarch agents can contain a Resistance outbreak.`
+	deadline
+	passengers 8 23
+	to offer
+		random < 15
+		has "license: Heliarch"
+	source
+		government "Heliarch"
+	stopover
+		distance 6 10
```

Comment:
idk why it even was a stopover, changed it to `destination`, added `not attributes "coalition station"` (hoping that works) to prevent it from rolling those as a destination